e2e-tests/playwright/e2e/plugins/application-provider-listener.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test } from "@playwright/test";
+import { UIhelper } from "../../utils/ui-helper";
+import { Common } from "../../utils/common";
+import { HomePage } from "../../support/pages/home-page";
+
+test.describe("Test ApplicationProvider", () => {
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    const common = new Common(page);
+    uiHelper = new UIhelper(page);
+    await common.loginAsGuest();
+  });
+
+  test("Verify that the TestPage is rendered", async ({ page }) => {
+    await page.goto("/application-provider-test-page");
+    await uiHelper.verifyText("application/provider TestPage");
+    await uiHelper.verifyText(
+      "This card will work only if you register the TestProviderOne and TestProviderTwo correctly.",
+    );
+  });
+});
+
+test.describe("Test ApplicationListener", () => {
```

Comment:
Can we split this into two files?

application-provider.spec.ts and application-listener.spec.ts? They are aligned, of course but I see it as two features.